onnxruntime/python/tools/transformers/models/whisper/requirements.txt

```diff
@@ -0,0 +1,19 @@
+# NOTE: In addition to the below packages, you will need to install `ffmpeg` on your machine.
+# Visit https://ffmpeg.org/ for details. You can also install it natively using package managers.
+# Ex:
+# Linux: sudo apt-get install ffmpeg
+# MacOS: sudo brew install ffmpeg
+# Windows: Download from website
+
+torch>=1.13.0
+transformers>=4.24.0
+openai-whisper
+ffmpeg-python
+datasets
+soundfile
+librosa
+optimum
+onnxruntime-gpu>=1.17.0
```

Comment:
1.17 cannot run new attributes added by this PR. Maybe add some guard in script to check that the version >= 1.18 when those new attributes exist in onnx.